mobile/src/board/pages.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactElement, useContext } from "react";
+import { useLoaderData, LoaderFunctionArgs } from "react-router-dom";
+
+import { Board } from "./entity";
+import BoardView from "./components/BoardView";
+import { useBoard } from "./hooks";
+
+export function GetById(): ReactElement {
+  const initialState = useLoaderData() as Board;
+  const board = useBoard(initialState);
```

Comment:
This isn't very readable. I understand that the hook subscribes to the board updates. Why not write it as:

```jsx
const initialBoard = useLoaderData<Board>();
const board = useUpdateBoardOnRemoteChange(initialBoard);
```